rules/config.js

```diff
@@ -56,6 +57,14 @@ module.exports = {
     'prefer-const': 2,
     'prefer-arrow-callback': 2,
     'strict': [ 2, 'never' ],
-    'no-const-assign': 2
+    'no-const-assign': 2,
+
+    // Node
+    'node/no-exports-assign': 'error',
+    'node/no-extraneous-require': 'error',
+    'node/no-missing-require': 'error',
+    'node/process-exit-as-throw': 'error', // This doesn't produce any errors
+    'node/no-deprecated-api': 'warn',
+    'node/shebang': 'warn'
```

Comment:
I think if we believe in the usefulness of these rules we should be willing to commit to calling them `error`. Otherwise it's just a check engine light.

Reply:
Updated to `error`